Automattic/wp-calypso
# New layout to support evergreen campaigns / weekly budget

Related to #



## Proposed Changes
**Before**
![Screenshot 2024-03-11 at 15 57 48](https://github.com/Automattic/wp-calypso/assets/6440498/8d9778c8-1bb2-4e66-a644-c3213cdea50b)

**After (left traditional, right evergreen)**
![Screenshot 2024-03-11 at 15 57 08](https://github.com/Automattic/wp-calypso/assets/6440498/f9d14d0a-c008-4259-9245-53fb91974718)

**Design**
![Screenshot 2024-03-12 at 16 10 18](https://github.com/Automattic/wp-calypso/assets/6440498/b29e7cb1-280c-4093-9768-bc64cc5f949a)


## Testing Instructions

Check the page using the live  link below, this should let you check a traditional campaign.

To test an evergreen campaign, you will need to load calypso locally, then force `is_evergreen` to `true`, see this commit for an example:
https://github.com/Automattic/wp-calypso/pull/88379/commits/ee1428a577ed6590efc33324c772feffb29b5d9c


Note: 

- "Duration so far" will only show when campaign is active too.
- Weekly spend summary will be in a follow on PR, I need to add this info to the campaign endpoint
- I've hidden stats when evergreen campaign is scheduled, Since "spent so far" values are all empty and so is impressions
![Screenshot 2024-03-12 at 17 33 11](https://github.com/Automattic/wp-calypso/assets/6440498/375ecd71-d1ac-46fc-b53a-64848945f9bf)


## Pre-merge Checklist
